doc/manual/contracts.md

```diff
@@ -567,8 +565,7 @@ contracts combinators are lazy too.
 
 Imagine we want to write a contract similar to `{_ : Bool}`, that is a
 dictionary of booleans, but we also want keys to be number literals (although
-represented as strings). A valid value could look like `{"1": true, "2": false,
-"10": true}`. If we used boolean predicates as the default for contracts, it
+represented as strings). A valid value could look like `{"1": true, "2": false, "10": true}`. If we used boolean predicates as the default for contracts, it
 would be impossible to make it lazy: as soon as your contract is called, you
 would need to produce a `true` or `false` answer, and checking that fields are
 all `Bool` would force their evaluation.
```

Comment:
Same (80 character wrapping)

```suggestion
represented as strings). A valid value could look like
`{"1": true, "2": false, "10": true}`. If we used boolean predicates as the
default for contracts, it would be impossible to make it lazy: as soon as your
contract is called, you would need to produce a `true` or `false` answer, and
checking that fields are all `Bool` would force their evaluation.
```